src/dmd/root/filename.d

```diff
@@ -828,7 +828,7 @@ version(Windows)
                                                 null /*filePartBuffer*/);
             if (pathLength == 0)
             {
-                return F(""w.ptr);
+                return F((wchar*).init);
```

Comment:
Why not simply `null`?

Reply:
Good point.

Reply:
Because type inference doesn't work otherwise and it broke the build. Reverting.